### PR TITLE
fix(ext/crypto): adjust `getRandomValues` types

### LINKED ISSUE #14710: Crypto.getRandomValues definition doesn't include BigUint64Array
In **lib.deno.crypto.d.ts** `Crypto.getRandomValues` does not include `BigUint64Array` despite v8 supporting this type with `getRandomValues`.

    declare interface Crypto {
      readonly subtle: SubtleCrypto;
      getRandomValues<
        T extends
          | Int8Array
          | Int16Array
          | Int32Array
          | Uint8Array
          | Uint16Array
          | Uint32Array
          | Uint8ClampedArray
          | Float32Array
          | Float64Array
          | DataView
          | null,
      >(
        array: T,
      ): T;
      randomUUID(): string;
    }

### PR DESCRIPTION
**Pull request by mrkldshv**
Add `BigInt64Array`, `BigUint64Array` and remove `Float32Array`, `Float64Array` types from crypto `getRandomValues`.
Closes #14710.

### CODE REVIEW

File: ext/crypto/lib.deno_crypto.d.ts [hunk 1]
```diff
@@ -326,6 +326,7 @@ declare interface Crypto {
       | Uint16Array
       | Uint32Array
       | Uint8ClampedArray
+      | BigUint64Array
```
Comment (darvesh) [inline ext/crypto/lib.deno_crypto.d.ts:329]: You missed `BigInt64Array`

Comment (darvesh) [reply ext/crypto/lib.deno_crypto.d.ts:329]: image

Spec mentions that `getRandomValues` only takes integer-based `TypedArray`, does removing `Float64Array` and `Float32Array` can be done in this PR itself?

Comment (mrkldshv) [reply ext/crypto/lib.deno_crypto.d.ts:329]: Good catch! Sure, we can apply these changes. What do you think? @lucacasonato

Comment (lucacasonato) [reply ext/crypto/lib.deno_crypto.d.ts:329]: SGTM

Comment (mrkldshv) [reply ext/crypto/lib.deno_crypto.d.ts:330]: Should be fixed now

**Review by bartlomieju — APPROVED** [review]
LGTM, but I'll wait for @lucacasonato approval before merging.

**Review by lucacasonato — APPROVED** [review]
LGTM. Thanks @mrkldshv!

**Review by littledivy — APPROVED** [review]

**Pull request merged**